src/routes/login/+page.svelte

```diff
@@ -25,10 +25,51 @@ for submission to the network.
     import { goto } from '$app/navigation'
     import { errorMessage } from '$lib/stores/alertsStore'
     import { walletStore } from '$lib/stores/walletStore'
+    import { Keypair } from 'stellar-sdk'
+    import { allowAllModules, StellarWalletsKit, WalletNetwork, XBULL_ID } from '@creit.tech/stellar-wallets-kit'
+    import { fundWithFriendbot } from '$lib/stellar/horizonQueries'
 
     // Define some component variables that will be used throughout the page
+    let keypair = Keypair.random()
+    $: publicKey = keypair.publicKey()
+    $: secretKey = keypair.secret()
+    let showSecret = false
     let pincode = ''
 
+         // Initialize Stellar Wallet Kit
+         const kit = new StellarWalletsKit({
+        network: WalletNetwork.TESTNET,
+        selectedWalletId: XBULL_ID,
+        modules: allowAllModules(),
+    })
+
+
+
+    const connectWallet = async () => {
```

Comment:
This function can be simplified. We don't need to check for registration status or do any redirection _here_ on the page, that's handled in the `+page.js`'s `load()` function. Same is true for the use of `fundWithFriendbot()`, that's handled by the signup route (and soon will be handled for logins, too: #7).

Really, all you're wanting to check is if the address they're providing via the wallet connection matches with the wallet that exists as the registered `walletStore.publicKey`.

Reply:
simplified the connect wallet function for both login and signup